howto/port/install-as-system-app.md

```diff
@@ -0,0 +1,70 @@
+Anbox Cloud allows user applications to be installed as Android system apps. This can be done by using the `aam install-system-app` command.
+
+### Make a system app
+To be a system app running in the system process of Android system, you must add the attribute `android:sharedUserId="android.uid.system"` into the `<manifest>` or `<application>` tag in the AndroidManifest.xml file of your Android project.
```

Comment:
Is it important whether you put it in the manifest tag or the application tag?
If it doesn't really matter, we should mention only one of them. If there is a difference, we need to explain it.

Reply:
It matters in the way that those are only two tags that the attribute `android:sharedUserId` should be put into. Going through AOSP code base, looks like the attribute is put into `<manifest>` tag in the source tree of almost all the system apps. I think we should follow.

Reply:
It matters in the way that those are only two tags that the attribute `android:sharedUserId` should be put into. Going through AOSP code base, looks like the attribute is put into `<manifest>` tag in the source tree of almost all the system apps. I think we should follow.